lib/settings_base.py

```diff
@@ -600,6 +600,7 @@ def JINJA_CONFIG():
             'js/zamboni/buttons.js',
             'js/zamboni/tabs.js',
             'js/common/keys.js',
+            'js/common-min.js',
```

Comment:
No, those are both generated files. They're both generated from the files in this list, so including them here is basically recursive. If adding it here fixes the problem, the issue is most likely the order of inclusion.